backend/e2e-test/routes/v2/service-token.spec.ts

```diff
@@ -0,0 +1,507 @@
+import crypto from "node:crypto";
+
+import { SecretType, TSecrets } from "@app/db/schemas";
+import { decryptSecret, encryptSecret, getUserPrivateKey, seedData1 } from "@app/db/seed-data";
+import { decryptAsymmetric, decryptSymmetric128BitHexKeyUTF8, encryptSymmetric128BitHexKeyUTF8 } from "@app/lib/crypto";
+
+const createServiceToken = async (
```

Comment:
Yah i am also still understanding how i should name these ones. I just put it based on api versioning for now. 

The issue with above naming convention is its not easy to find whether one api test has been written or not. The current structure is simple actually all the test files are the tests for the same api in `router` folder inside `server` folder. So it's very easy to find test endpoint of a service. 

Now regarding your confusion along identity spec has both creating identity+ universal auth. Its because as you mentioned in another comment. Each describe is independent. So for identity token testing, to get access i need to trigger identity universal auth based login setup. 

Same goes for service token testing for secrets. I would need to create service token to do the testing right, to get access to secrets.